src/org/usfirst/frc/team4571/robot/RobotJoystick.java

```diff
@@ -0,0 +1,86 @@
+package org.usfirst.frc.team4571.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class RobotJoystick extends Joystick {
+		private Button button_1;
+		private Button button_2;

```

Comment:
change the name of all the buttons from button_x to buttonX. so button_1 becomes button1